Application/EdFi.Common/Configuration/IConfigValueProvider.cs

```diff
@@ -1,12 +1,15 @@
 // SPDX-License-Identifier: Apache-2.0
+#if NETFRAMEWORK
```

Comment:
this is in the wrong spot it should be above the namespace

Reply:
Moved above namespace